music_rnn/src/model.ts

```diff
@@ -15,55 +15,46 @@
  * =============================================================================
  */
 
-import * as dl from 'deeplearn';
 import * as magenta from '@magenta/core';
-import * as data from './data';
-import { INoteSequence } from '@magenta/core';
-
-// tslint:disable-next-line:max-line-length
-const CHECKPOINT_URL = "https://storage.googleapis.com/download.magenta.tensorflow.org/models/music_rnn/dljs/basic_rnn/";
-
-const DEFAULT_MIN_NOTE = 48;
-const DEFAULT_MAX_NOTE = 83;
+import tf = magenta.tf;
+import { isNullOrUndefined } from 'util';
```

Comment:
where is util coming from?

Reply:
not sure...it must be a dependency somewhere? i found in in autocomplete

Reply:
Ah, looks like it's a node thing. It will work locally, but I think we need to add a dependency on the 'util' package for bundling purposes.